src/components/hex-map/index.js

```diff
@@ -59,6 +59,9 @@ const mapStateToProps = createStructuredSelector({
   paintRegion: currentPaintRegion,
   paintRegionId: layerRegionPaintSelector,
   layerHexes: visibleLayerHexColors,
+  showEntityCount: state => state.settings.showEntityCount,
+  showSystemName: state => state.settings.showSystemName,
+  showSystemNumber: state => state.settings.showSystemNumber,
```

Comment:
These should be defined in the base selectors file.

Reply:
Fixed in Irame/sectors-without-number@3e8c22179164e4aa335f4a4e3daa4b3a158d39d8